README.md

```diff
@@ -15,16 +18,15 @@ This section documents the assumptions that `prodmgr` makes about that image.
 
 The install utility image:
```

Comment:
I think we should be extra clear here as well that this is just the fallback (legacy) product-specific install utility image. I don't want other product teams thinking they need to implement one of these to get the deletion functionality.

Reply:
"If you have an important point to make, don't try to be subtle or clever. Use a pile driver. Hit the point once. Then come back and hit it again." ~ Winston Churchill

Point driven home in the README.